README.md

```diff
@@ -1 +1,26 @@
-# HDC
\ No newline at end of file
+Implementation of Hyper Dimensional Computing (HDC) using FPGA
+========
+
+Contributors
+--------
+* [Professor Sitao Huang](https://engineering.uci.edu/users/sitao-huang)
+* [Joseph Prince](https://github.com/josephdprince/)
+* [Hong Dong](https://github.com/hongdong1220)
+* Kyle Van
+* Brad Lacy
```

Comment:
Brad is not a contributor on this project

Reply:
We have him on the paper, so I am including him